publisher/publisher.go

```diff
@@ -0,0 +1,57 @@
+package publisher
+
+import (
+	"fmt"
+
+	"github.com/screwdriver-cd/sd-cmd/config"
+	"github.com/screwdriver-cd/sd-cmd/screwdriver/api"
+	"github.com/screwdriver-cd/sd-cmd/util"
+)
+
+func init() {
+	config.LoadConfig()
+}
+
+// Publisher is a type to publish sdapi and sdstore.
+// It receive strings which input by user.
+// If -f option is valid, yml file will be loaded to commandSpec struct.
```

Comment:
Screwdriver.cd uses "yaml" expression.

Reply:
Fixed. Do you think all of ```yml``` expression like function name(ex. LoadYml) or filename(ex. testdata/command_spec.yml) are also should be renamed?